packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed 
+* Fixed am issue to prevent multiple dropdowns from opening in mobile and move focus to select menu when opened via toggle button.
```

Comment:
Small typo, 

`am` -> `an`

This entry should probably be split into two entries for each of the different fixes

Reply:
Oopps Fixed here 3c82c03